challenges/2021_cloud_base_height/07_final_torch_eval.ipynb

```diff
@@ -0,0 +1,939 @@
+{
```

Comment:
<p>Might be worth emphasing why there is this additional evaluation notebook and what is different or similar compared to the other one.</p><p> </p><p>Maybe also include the number of the notebook in the title so people know they are looking at the right notebook?</p><p>Also what happened to notebook number 6?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>